src/Microsoft.Diagnostics.Monitoring.Options/OptionsDisplayStrings.resx

```diff
@@ -811,6 +811,30 @@
     <value>The type of metrics this provider consumes</value>
     <comment>The description provided for the MetricType parameter on MetricProvider.</comment>
   </data>
+  <data name="DisplayAttributeDescription_SystemDiagnosticsMetricsOptions_InstrumentName" xml:space="preserve">
+    <value>The name of the instrument to monitor.</value>
+    <comment>The description provided for the InstrumentName parameter on SystemDiagnosticsMetricsOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_SystemDiagnosticsMetricsOptions_GreaterThan" xml:space="preserve">
+    <value>The threshold level the instrument must maintain (or higher) for the specified duration. Either GreaterThan or LessThan (or both) must be specified.</value>
+    <comment>The description provided for the GreaterThan parameter on SystemDiagnosticsMetricsOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_SystemDiagnosticsMetricsOptions_HistogramPercentile" xml:space="preserve">
+    <value>When monitoring a histogram, this dictates which percentile to compare against using the value in GreaterThan/LessThan.</value>
+    <comment>The description provided for the HistogramPercentile parameter on SystemDiagnosticsMetricsOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_SystemDiagnosticsMetricsOptions_LessThan" xml:space="preserve">
+    <value>The threshold level the instrument must maintain (or lower) for the specified duration. Either GreaterThan or LessThan (or both) must be specified.</value>
+    <comment>The description provided for the LessThan parameter on SystemDiagnosticsMetricsOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_SystemDiagnosticsMetricsOptions_ProviderName" xml:space="preserve">
```

Comment:
Either have this as MeterName, or have InstrumentName be CounterName (not half and half)

Reply:
I've updated this to use MeterName and InstrumentName, while silently accepting ProviderName/CounterName for anyone who tries to follow the existing format

Reply:
I think having duplicate options that do the same thing is more confusing than just changing the name; I think users who are familiar with System.Diagnostics.Metrics will find MeterName and InstrumentName very clear, even if not consistent with the previous trigger names.

Reply:
I think where this becomes interesting is that users are currently using the ProviderName/CounterName pattern to specify `SystemDiagnosticsMetrics` when listing custom providers (for `Metrics` in configuration) - which means in one part of config you'll call your meter a provider, and then you'll need to remember to call it a meter in the collection rule. That being said, it feels odd to make the official options incorrect (given that we know it's an `SDM` trigger, provider/counter aren't the correct terms, and I don't know if we want to establish a precedent of officially using the wrong name). This was intended to be a compromise that reduces the need for user education about the distinction, while keeping the provider/counter option undocumented so people don't think MeterName/ProviderName or InstrumentName/CounterName are different and both need to be configured with different values.

Basically, if a user read the documentation, they'd follow the "correct" naming scheme, but we're also not going to force users who don't read docs to understand the nuance since we know what they meant. I'm open to changing this if people feel strongly one way or another though.